__tests__/app.test.js

```diff
@@ -182,3 +182,77 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  it("Posts a new comment to the specific article and returns the comment", () => {
+    const comment = {
+      username: "butter_bridge",
+      body: "comment",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(comment)
+      .expect(201)
+      .then(({ body: { comment } }) => {
+        expect(comment).toMatchObject({
+          comment_id: 19,
+          body: "comment",
+          article_id: 1,
+          author: "butter_bridge",
+          votes: 0,
+          created_at: expect.any(String),
+        });
+      });
+  });
+  it("Returns an error if an invalid comment object is sent", () => {
+    const invalidComments = [
+      {
+        invalidKey: "error",
+      },
+      {
+        invalidKey: "error",
+        body: "comment",
+      },
+      {
+        invalidKey: "error",
```

Comment:
Although the above 2 comments should have an error, this last comment 
```
{
  invalidKey: "error",
 body: "comment",
 username: "butter_bridge",
  }
```
could still be a 201. The extra property of `invalidKey` should be ignored, but the comment itself still posted.